src/Components/Footer/Footer.css

```diff
@@ -0,0 +1,18 @@
+.footer {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  height: 80px; /*sets the height of the footer, can adjust if we want it larger/smaller */
+  background-color: #F7F7F9;
+  position: absolute;
+  bottom: 0;
+  width: 100%;
```

Comment:
I'd also add margin-top: auto so that the footer stays at the bottom of the page.